oci/layout/oci_dest_test.go

```diff
@@ -58,3 +58,19 @@ func TestPutBlobDigestFailure(t *testing.T) {
+
+	err = ociDest.PutManifest([]byte(m))
+	require.Error(t, err)
+	assert.Contains(t, `Unrecognized manifest media type: "application/vnd.docker.distribution.manifest.v1+prettyjws"`, err.Error())
```

Comment:
I suppose this identifies the underlying bug in https://github.com/projectatomic/skopeo/issues/198 : we guess schema1 manifests as a MIME type which isn’t recognized.

OTOH as soon as we will recognize that MIME type, this test will permanently break…